packages/oasis-actions/src/strategies/aave/close/close.ts

```diff
@@ -346,11 +348,12 @@ async function generateTransition(
   // We need to calculate a fee from the total locked collateral
   // Then convert this amount into the debt token
   const actualMarketPriceWithSlippage = swapData.fromTokenAmount.div(swapData.minToTokenAmount)
+  const fee = feeResolver(args.collateralToken.symbol, args.debtToken.symbol)
   const postSwapFee =
     collectFeeFrom === 'targetToken'
       ? calculateFee(
           dependencies.currentPosition.collateral.amount.div(actualMarketPriceWithSlippage),
-          new BigNumber(DEFAULT_FEE),
+          new BigNumber(fee),
```

Comment:
isn't `fee` already of type `BigNumber` ?

Reply:
Sorted.